api/src/cache.ts

```diff
@@ -65,7 +65,11 @@ export function getCache(): {
 	}
 
 	if (localSchemaCache === null) {
-		localSchemaCache = getKeyvInstance('memory', getMilliseconds(env['CACHE_SYSTEM_TTL']), '_schema');
+		localSchemaCache = getKeyvInstance('memory', getMilliseconds(env['CACHE_SYSTEM_TTL']), '_schema', {
+			serialize: (v: any) => Object.freeze(v),
```

Comment:
Although we do not seem to directly modify the schema we do expose it to extensions. As such this is potentially a breaking change as freezing the object will cause unexpected results for anyone who is modifying it.

Additionally this change requires all contributors to be aware that the schema is frozen when working in the codebase